### PR TITLE
fixed algolia search page

### PR DESCRIPTION
**Pull request by cdkini**
Duplicate of https://github.com/great-expectations/great_expectations/pull/4160 due to GitHub merge issues.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -145,13 +168,20 @@ module.exports = {
...
+        //   current: {
+        //     label: 'docs',
+        //     path: 'docs'
+        //   },
+        //   '0.13.9': {
+        //     label: '0.13.9-docs',
+        //     path: '0.13.9'
+        //   }
+        // }
+      }
```
Comment (Shinnnyshinshin) [inline docusaurus.config.js:176]:
```suggestion
      })
```

I just had a peek at the `deploy-preview` and I think this might be missing :)

Comment (cdkini) [reply docusaurus.config.js:176]: hmm that doesn't seem to fix it (my editor says that the file is valid JS)

**Comment by kyleaton** [comment]
@cdkini @Shinnnyshinshin I've added @devanshdixit from Novvum to our GitHub org so he can work directly with your code here. He'll be picking it up from here!

File: package.json [hunk 2]
```diff
@@ -15,12 +15,14 @@
     "lint": "standard --fix"
   },
   "dependencies": {
-    "@cmfcmf/docusaurus-search-local": "^0.10.0",
-    "@docusaurus/core": "^2.0.0-beta.20",
-    "@docusaurus/plugin-google-gtag": "^2.0.0-beta.20",
-    "@docusaurus/plugin-sitemap": "^2.0.0-beta.20",
-    "@docusaurus/preset-classic": "^2.0.0-beta.20",
-    "@docusaurus/theme-search-algolia": "^2.0.0-beta.20",
+    "@cmfcmf/docusaurus-search-local": "^0.11.0",
```
Comment (donaldheppner) [inline package.json:18]: Why are we taking on alpha dependencies?

File: docusaurus.config.js [hunk 3]
```diff
@@ -150,8 +172,21 @@ module.exports = {
           trackingID: 'UA-138955219-1',
           // Optional fields.
           anonymizeIP: true // Should IPs be anonymized?
-        }
-      })
+        },
+        // lastVersion: 'current',
+        // versions: {
+        //   // Example configuration:
```
Comment (donaldheppner) [inline docusaurus.config.js]: Action required here?

**Review by donaldheppner — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (devanshdixit) [reply package.json:18]: I have updated the package version with a stable one @donaldheppner

File: docusaurus.config.js (hunk 3 above)
Comment (devanshdixit) [reply docusaurus.config.js]: This can be deleted @donaldheppner and I have done that

**Review by donaldheppner — APPROVED** [review]

**Pull request merged**